### PR TITLE
[Enhancement](drop) drop multi tables in one drop stmt

### PR DESCRIPTION
**Pull request by zzzzzzzs**
# Proposed changes

Issue Number: https://github.com/apache/doris/issues/19418

## Problem summary

Support drop multi tables in one drop stmt:

```sql
drop table t1, t2;
```

## Checklist(Required)

* [x] Does it affect the original behavior
* [x] Has unit tests been added
* [x] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zzzzzzzs** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.26 seconds
 stream load tsv:          430 seconds loaded 74807831229 Bytes, about 165 MB/s
 stream load json:         24 seconds loaded 2358488459 Bytes, about 93 MB/s
 stream load orc:          61 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230514115338_clickbench_pr_143392.html

**Comment by dataroaring** [comment]
please add a regression case under regression-test.

**Comment by zzzzzzzs** [comment]
> please add a regression case under regression-test.

added regression-test

**Comment by stalary** [comment]
I have a question, what happens if a table fails while deleting multiple tables?

**Comment by zzzzzzzs** [comment]
> I have a question, what happens if some table fails while deleting multiple tables?

Currently, if dropping a table fails, the program will terminate the drop operation. For example, if you run `drop table t1,t2,t3;` and an error occurs while dropping table t2, both t2 and t3 tables will stop the drop operation.

File: fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java [hunk 1]
```diff
@@ -807,67 +807,70 @@ public void replayRenameDatabase(String dbName, String newDbName) {
 
     // Drop table
     public void dropTable(DropTableStmt stmt) throws DdlException {
-        String dbName = stmt.getDbName();
-        String tableName = stmt.getTableName();
+        for (TableName tbl : stmt.getTableNames()) {
+            String dbName = tbl.getDb();
+            String tableName = tbl.getTbl();
```
Comment (dataroaring) [inline fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java:812]: We'd better check before dropping and dropping multi tables in a transaction if possible.

**Pull request closed without merging**